pages/add-clinic.vue

```diff
@@ -5,65 +5,87 @@
         <h2>{{ $t("add-clinic.fillForm") }}</h2>
       </div>
       <v-container>
-        <div align="center" class="red--text text--darken-4">
+        <v-row class="red--text text--darken-4 mb-4 justify-center">
           <b>{{ $t("add-clinic.romajiOnly") }}</b>
-        </div>
+        </v-row>
         <v-row>
-          <v-col cols="12" sm="6">
-            <v-autocomplete
-              :items="prefectureList"
-              :label="$t('add-clinic.prefecture')"
-              v-model="prefecture"
-              required
-            ></v-autocomplete>
-          </v-col>
-          <v-col cols="12" sm="6" md="4">
-            <v-text-field
-              :label="$t('add-clinic.city')"
-              v-model="city"
-              required
-              :rules="cityRules"
-            ></v-text-field>
-          </v-col>
-          <v-col cols="12" sm="6" md="4">
-            <v-text-field
-              :label="$t('add-clinic.ward')"
-              v-model="ward"
-              required
-              :rules="wardRules"
-            ></v-text-field>
-          </v-col>
-          <v-col cols="12" sm="6" md="4">
-            <v-text-field
-              :label="$t('add-clinic.clinicName')"
-              v-model="name"
-              required
-              :rules="nameRules"
-            ></v-text-field>
-          </v-col>
-          <v-col cols="12" sm="6" md="4">
-            <v-text-field
-              :label="$t('add-clinic.websiteURL')"
-              v-model="website"
-              required
-              :rules="websiteRules"
-            ></v-text-field>
+          <v-autocomplete
+            :items="prefectureList"
+            :label="$t('add-clinic.prefecture')"
+            v-model="prefecture"
+            required
+            outlined
+          ></v-autocomplete>
+        </v-row>
+        <v-row>
+          <v-text-field
+            :label="$t('add-clinic.city')"
+            v-model="city"
+            required
+            filled
+            :rules="cityRules"
+          ></v-text-field>
+        </v-row>
+        <v-row>
+          <v-text-field
+            :label="$t('add-clinic.ward')"
+            v-model="ward"
+            required
+            filled
+            :rules="wardRules"
+          ></v-text-field>
+        </v-row>
+        <v-row>
+          <v-text-field
+            :label="$t('add-clinic.clinicName')"
+            v-model="name"
+            required
+            filled
+            :rules="nameRules"
+          ></v-text-field>
+        </v-row>
+        <v-row>
+          <v-text-field
+            :label="$t('add-clinic.websiteURL')"
+            v-model="website"
+            required
+            filled
+            :rules="websiteRules"
+          ></v-text-field>
+        </v-row>
+        <v-row>
+          <v-col>
+            <v-switch
+              v-model="voucher.required"
+              inset
+              :label="`Voucher required: ${voucher.text}`"
```

Comment:
Should the 'voucher required' also be in the translation?

Reply:
Yeah so all users know what the switch is for.